corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -163,7 +163,7 @@ def increment_app_version(app_doc):
 
     def get_app_ids(self, domain=None):
         return (list(get_deleted_app_ids(domain))
-                + list(get_all_app_ids(domain=domain, include_builds=True)))
+                + list(get_all_app_ids(domain=domain, include_builds=self.include_builds)))
```

Comment:
@AddisonDunn I am not quite sure I understand your question. I don't see any use of `self.include_builds`. So overriding it in any subclass does not do anything. I could set it to `True` in `AppMigrationCommandBase` to keep the current behavior. Or leave it out of this PR since I was not going to save the migration anyways.